### ISSUE TITLE
Missing Alt text of Media on Timeline

### ISSUE THREAD

**Issue by mdavismcd** [Daniel-KM/Omeka-S-module-Timeline#22]
Can the media's description (Alt text module) get pulled as the alt text, e.g., `alt="Davidson College Logo"` for the media on the timeline? The [Mapping Module](https://github.com/omeka-s-modules/Mapping) added this in [a recent commit](https://github.com/omeka-s-modules/Mapping/commit/1ec036c462a75d87f628b8167b7d3428115cc842) and has the following text. I've tried to implement it (and smaller snippets of the code) into different files in the Timeline configuration with no luck.

```
// Set the "media" object.
        $media = $item->primaryMedia();
        if ($media) {
            $event['media'] = [
                'url' => $media->thumbnailUrl('large'),
                'thumbnail' => $media->thumbnailUrl('medium'),
                'link' => $item->url(),
                'alt' => $media->altText(),
            ];
        }
```

Thanks so much!

**Comment by mdavismcd** [comment]
Hi, @Daniel-KM. I'm just checking in on this. Thanks!
